<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTaskStandardUserAccountWindow</class>
 <widget class="QMainWindow" name="AddNewTaskStandardUserAccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TRMS - Create New Task</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/images/TRMS-Logo-Without-Text.ico</normaloff>:/images/TRMS-Logo-Without-Text.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 192, 196);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create New Task</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Task Reminder Management System</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addTask_pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>380</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(248, 248, 248);</string>
     </property>
     <property name="text">
      <string>ADD TASK</string>
     </property>
    </widget>
    <widget class="QLabel" name="logo_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGO</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="title_lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Enter First Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="description_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="description_plainTextEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="category_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
    <widget class="QComboBox" name="category_comboBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 21, 0);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>170</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 21, 0);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>130</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 21, 0);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
